### DISCUSSION TITLE
Dont´t display images o any graphic...

### DISCUSSION THREAD

**Discussion by MarceloRojasOlivos** [pH7Software/pH7-Social-Dating-CMS#1059] [Troubleshooting]
![errors](https://user-images.githubusercontent.com/112295864/190935666-3b98bfac-1775-4edb-91cb-86afc9aac3cb.jpg)

After installing by softacoulus... every thing is like this... i have the correct php version... htaccess error?
Any idea?

**Comment by Shearsmith1337** [comment]
check your database limit
